WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/InPersonPaymentsMenuViewModel.swift

```diff
@@ -36,6 +43,8 @@ class InPersonPaymentsMenuViewModel: ObservableObject {
 
     var shouldAlwaysHideSetUpButtonOnAboutTapToPay: Bool = false
 
+    private(set) var orderViewModel: EditableOrderViewModel
```

Comment:
Question: Seeing we pass all other dependencies explicitly, should we inject the `EditableOrderViewModel` via constructor injection when the `InPersonPaymentsMenuViewModel` is created, rather than assigning to the property a new instance and avoid having an implicit dependency?

Reply:
Gotcha! (Thanks for the iOS17+ doc link!)